solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,96 @@
+##############################################################################
+# Input Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API Key."
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The region to provision all resources created by this example."
```

Comment:
this is not an example - also why do we even have region at all? s2s auth policies are not region scoped?